ext/libyuv.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5df76856..eae0b729 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -112,13 +112,13 @@ if(NOT MSVC)
+       ${ly_src_dir}/rotate_neon64.cc
+       ${ly_src_dir}/row_neon64.cc
+       ${ly_src_dir}/scale_neon64.cc)
+-    target_compile_options(${ly_lib_name}_neon64 PRIVATE -march=armv8-a+dotprod+i8mm)
++    target_compile_options(${ly_lib_name}_neon64 PRIVATE -march=armv8.2-a+dotprod+i8mm)
+     list(APPEND ly_lib_parts $<TARGET_OBJECTS:${ly_lib_name}_neon64>)
+ 
+     # Enable AArch64 SVE kernels.
+     add_library(${ly_lib_name}_sve OBJECT
+       ${ly_src_dir}/row_sve.cc)
+-    target_compile_options(${ly_lib_name}_sve PRIVATE -march=armv9-a+sve2)
++    target_compile_options(${ly_lib_name}_sve PRIVATE -march=armv8.5-a+sve2)
```

Comment:
@fdintino Frankie: In the review of the [libyuv CL](https://chromium-review.googlesource.com/c/libyuv/libyuv/+/6405834), George Stead suggested that we use `-march=armv8.5-a+i8mm+sve2` instead. Could you please test that and see if it works with gcc 10?

Reply:
Sure, though I won't be able to get to it until late this evening.

Reply:
`-march=armv8.5-a+i8mm+sve2` works fine.

Reply:
Frankie: Thanks a lot for verifying it works!